packages/bot-web-ui/src/app/app-content.jsx

```diff
@@ -20,12 +20,44 @@ const AppContent = observer(() => {
         ui: { is_dark_mode_on },
     } = RootStore;
     const DBotStores = useDBotStore();
-    const { app } = DBotStores;
+    const { app, transactions } = DBotStores;
     const { showDigitalOptionsMaltainvestError } = app;
 
     // TODO: Remove this when connect is removed completely
     const combinedStore = { ...DBotStores, core: { ...RootStore } };
 
+    const { recovered_transactions, recoverPendingContracts } = transactions;
+    const is_subscribed_to_msg_listener = React.useRef(false);
+    const msg_listener = React.useRef(null);
```

Comment:
We might want to use a more detailed name for this since the file we are adding it is a common file and not specific for open contracts.

Reply:
Wanted to include `proposal_open_contracts_msg_listener` but this one is listening for all the messages and from there we are only checking the type POC so kept it kind of generic. Should I stick to  `proposal_open_contracts_msg_listener`?